vllm/worker/model_runner.py

```diff
@@ -523,19 +544,52 @@ def _prepare_decode(
                 device=self.device,
             )
 
-        attn_metadata = self.attn_backend.make_metadata(
-            is_prompt=False,
-            prompt_lens=None,
-            prompt_lens_tensor=None,
-            max_subquery_len=None,
-            max_context_len=max_context_len,
-            max_prompt_len=None,
-            subquery_start_loc=None,
-            seq_start_loc=None,
-            context_lens=context_lens_tensor,
-            block_tables=block_tables,
-            use_cuda_graph=use_captured_graph,
-        )
+        flashinfer_wrapper = None
+        if self.attn_backend is FlashInferBackend:
+            if self.workspace_buffer is None:
+                self.workspace_buffer = torch.empty(16 * 1024 * 1024,
```

Comment:
QQ: what's the best practice buffer size here? can you add a comment in this code with a link?